src/main/resources/static/script.js

```diff
@@ -29,25 +28,26 @@ function initMap() {
         query: "'US national park'",
     };
 
-
+    let nationalParks = [];
```

Comment:
Does this array hold all the JSON object markers and info windows for all the national parks or just the 20 that are returned from the Places API? Just trying to understand the code as I'm reading it.